midp/gfx.js

```diff
@@ -940,8 +960,10 @@ var currentlyFocusedTextEditor;
         self.creator = null;
```

Comment:
> Don't really need to address in this PR, but should this be set to the null constant?

Nice catch, yes, it should, per the implementations of Graphics.getCreator and Graphics.setCreator. 8c73b14 makes the change.